File: server/utils/http_client.py
# import os
# import requests
# from dotenv import load_dotenv

# load_dotenv()

# BASE_URL = "https://sandboxapi.bitnob.co/api/v1"
# API_KEY = os.getenv("BITNOB_API_KEY")

# def bitnob_post(endpoint, data):
#     headers = {
#         "Accept": "application/json",
#         "Content-Type": "application/json",
#         "Authorization": f"Bearer {API_KEY}"
#     }
#     url = f"{BASE_URL}{endpoint}"
#     response = requests.post(url, json=data, headers=headers)
#     response.raise_for_status()
#     return response.json()

File: bitnob/utils/httpClient.js
require('dotenv').config();
const axios = require('axios');

const httpClient = axios.create({
  baseURL: 'https://sandboxapi.bitnob.co/api/v1',
  headers: {
    Accept: 'application/json',
    'Content-Type': 'application/json',
    Authorization: `Bearer ${process.env.BITNOB_API_KEY}`
  }
});

module.exports = httpClient;